definitions/divorce/json/State/State.json

```diff
@@ -442,7 +442,7 @@
     "Name": "Awaiting General Referral Payment",
     "Description": "Awaiting General Referral Payment",
     "DisplayOrder": 35,
-    "TitleDisplay": "${[CASE_REFERENCE]}: ${D8PetitionerLastName} vs ${D8RespondentLastName}"
+    "TitleDisplay": "${[CASE_REFERENCE]}: ${D8PetitionerLastName} vs ${D8RespondentLastName}"
```

Comment:
what has changed?

Reply:
if you increase screen resolution to 4k it will become obvious it is the space between case reference and petitioner name to NBSP to unify state names.